### ISSUE TITLE
Cloudmin: Host Systems/KVM Host Systems missing closing body tag

### ISSUE THREAD

**Issue by qooob** [webmin/webmin#825]
Jamie, hi.

I've been testing Cloudmin before the release and found that mentioned `Host Systems/KVM Host Systems` doesn't have closing body tag, which makes Authentic fail in loading the page.

This is how the Virtualmin Framed Theme displays it, so you'd know where to look.
![screenshot from 2018-02-22 11-55-18](https://user-images.githubusercontent.com/4426533/36528842-8983cb68-17c7-11e8-85e3-6d24bddda06e.png)

**Comment by qooob** [comment]
I finally added a patch for the theme to workout such broken pages. It will work with this fix that is going to 19.09.2 within an hour.
https://github.com/qooob/authentic-theme/commit/fb758cdcff462a4b2b3b8cd57c481f5207388722

**Comment by qooob** [comment]
Example:

![screenshot from 2018-02-22 17-27-36](https://user-images.githubusercontent.com/4426533/36543768-ce26d020-17f5-11e8-8292-cba30f9b673e.png)

This is the global patch, so it will affect all possibly broken pages.

**Comment by gnadelwartz** [comment]
would it be possible to report if a broken page is detected to fix it?

**Comment by qooob** [comment]
Yes, but as a debug option. I will add it to the todo for 19.10.

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
